addon/result_iterator.cc

```diff
@@ -182,21 +191,21 @@ Napi::Value ResultIterator::getCellValue(Napi::Env env, duckdb::idx_t col_idx) {
       throw runtime_error("expected int64 for timestamp");
     }
     int64_t tval = val.GetValue<int64_t>();
-    int64_t date = Epoch(GetDate(tval)) * 1000;
-    int32_t time = GetTime(tval);
-    return Napi::Number::New(env, date + time);
-  }
-  case duckdb::LogicalTypeId::DATE: {
-    if (result->types[col_idx].InternalType() != duckdb::PhysicalType::INT32) {
-      throw runtime_error("expected int32 for date");
-    }
-    return Napi::Number::New(env, Epoch(val.GetValue<int32_t>()) * 1000);
-  }
+    return Napi::Number::New(env, tval / 1000);
+  }
+  // case duckdb::LogicalTypeId::DATE: {
+  //   cout << "DATE" << endl;
+  //   if (result->types[col_idx].InternalType() != duckdb::PhysicalType::INT32)
+  //   {
+  //     throw runtime_error("expected int32 for date");
+  //   }
+  //   return Napi::Number::New(env, Epoch(val.GetValue<int32_t>()) * 1000);
+  // }
```

Comment:
We're going with the string representation for now, right? Do we still need this (commented) code?

Reply:
Yes we are. I left it here in case we ever go back to it, but you're right, may as well just have it in git history